src/ORM/Behavior/TimestampBehavior.php

```diff
@@ -130,19 +130,23 @@ public function implementedEvents()
      * If an explicit date time is passed, the config option `refreshTimestamp` is
      * automatically set to false.
      *
-     * @param \DateTime|null $ts Timestamp
+     * @param \DateTimeInterface|null $ts Timestamp
      * @param bool $refreshTimestamp If true timestamp is refreshed.
-     * @return \DateTime
+     * @return \DateTimeInterface
      */
-    public function timestamp(DateTime $ts = null, $refreshTimestamp = false)
+    public function timestamp(DateTimeInterface $ts = null, $refreshTimestamp = false)
```

Comment:
Strictly speaking, we cannot change signatures in minor releases. It may be better to change from master to 3.next.